pkgs/applications/misc/hollywood/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, makeWrapper, lib, coreutils, apg, atop, bmon, cmatrix
+, pygments, moreutils, util-linux, jp2a, man, mplayer, openssh, tree, mlocate
+, findutils, ccze, ncurses, python3, wget, libcaca, newsboat, rsstail, w3m
+, ticker, tmux, byobu }:
+let
+  pathDeps = [
+    tmux
+    coreutils
+    ncurses
+    jp2a
+    mlocate
+    apg
+    atop
+    bmon
+    cmatrix
+    pygments
+    moreutils
+    util-linux
+    jp2a
+    man
+    mplayer
+    openssh
+    tree
+    findutils
+    ccze
+  ];
+in stdenv.mkDerivation {
```

Comment:
Why aren't you using  `rec`?

Reply:
Because I didn’t reference anything from the attrset.